pkg/services/playlist/playlistimpl/object_store.go

```diff
@@ -0,0 +1,179 @@
+package playlistimpl
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/playlist"
+	"github.com/grafana/grafana/pkg/services/sqlstore/session"
+	objectstore "github.com/grafana/grafana/pkg/services/store"
+	"github.com/grafana/grafana/pkg/services/store/object"
+	"github.com/grafana/grafana/pkg/services/user"
+)
+
+// This is a playlist implementation that will:
+// 1. CREATE/UPDATE/DELETE everythign with existing direct SQL store
+// 2. CREATE/UPDATE/DELETE same items to the object store
+// 3. Use the object store for all read operations
+// This givs us a safe test bed to work with the store but still roll back without any lost work
+type objectStoreImpl struct {
+	sess   *session.SessionDB
+	backup *Service
+	server object.ObjectStoreServer
```

Comment:
not a big deal, but my initial feedback when reading this is that it might be easier to follow if these were named sqlStore and objectStore instead of backup and server

Reply:
👍 -- makes sense... the name "objectStore" still gets me since it is also SQL 🤦 

I will update